docs/.vitepress/theme/components/Accordion.vue

```diff
@@ -11,7 +11,7 @@ const props = withDefaults(defineProps<{
 </script>
 
 <template>
-<Component :is="props.tag" class="border-b border-opacity-20 border-gold py-10 px-4 p2 text-gray-06">
+<Component :is="props.tag" class="border-b border-gold/20 !py-10 !px-4 p2 !text-gray-06">
```

Comment:
Is this supposed to be <component>  with lowercase?

Reply:
hmm not sure what exactly you're talking about (the tag name, the class, or something else) but everything is in the correct case: `Component`, capital C as per common practice with custom element / components (not required, just a practice that most dev follows, including me), and the css classes all lowercase, hyphen separated as per xhtml / html5 standard, not required either, just common.